test-data/unit/fixtures/args.pyi

```diff
@@ -19,7 +19,9 @@ class type:
     def __call__(self, *args: Any, **kwargs: Any) -> Any: pass
 
 class tuple(Iterable[Tco], Generic[Tco]): pass
-class dict(Generic[T, S]): pass
+
+class dict(Iterable[T], Mapping[T, S], Generic[T, S]):
+    pass
```

Comment:
nit: it seems like all other fixtures have `pass` on the same line as the class declaration. I would try to be consistent.